code/modules/research/designs/weapon_designs.dm

```diff
@@ -63,10 +63,65 @@
 	req_tech = list("combat" = 4, "materials" = 3, "powerstorage" = 3)
 	build_type = PROTOLATHE
 	materials = list(MAT_METAL = 10000, MAT_GLASS = 2000, MAT_DIAMOND = 2000)
+	build_path = /obj/item/weapon/gun/energy/laser/hypercannon
+	locked = 1
+	category = list("Weapons")
+
+/datum/design/laserhypercannon
+	name = "Laser Hypercannon"
+	desc = "A heavy duty laser cannon that increases in damage the further the target."
+	id = "laserhypercannon"
+	req_tech = list("combat" = 5, "materials" = 4, "powerstorage" = 6)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 10000, MAT_GLASS = 2000, MAT_URANIUM = 1000, MAT_DIAMOND = 1000, MAT_PLASMA = 3000, MAT_GOLD = 3000)
```

Comment:
Pretty cheap for a cannon that can deal 100 damage, should need atleast 10 sheet of everything considering that 2000 = 1 sheet